### ISSUE TITLE
Fix tests.py bug with updated models.py

### ISSUE THREAD

**Issue by mccrosno** [Newt-Tea/Intro_SE_WebApp_Group1#58]
TL;DR - Tests.py user test cannot locate subclass object from models.py in DB, need help/working on fix.

Committed changes in this branch with an updated tests.py file with Product and User tests to verify that they are being added into the database, with the goal of making sure that the models can be added into the database and verify that there is a way of being able to both create new objects of each model type as well as view their properties.

I'm uploading this as an issue to see if anyone else can find a way to create an Admin, Seller, or Buyer object from the models.py file and then locate that object in the database based off their primary key, the userID CharField in models.py.
